Signal/src/ViewControllers/OWSTableViewController.m

```diff
@@ -196,6 +197,20 @@ - (void)loadView
     [self.tableView registerClass:[UITableViewCell class] forCellReuseIdentifier:kOWSTableCellIdentifier];
 }
 
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+
+    [self.navigationController.navigationBar setTranslucent:NO];
+}
+
+- (void)viewWillAppear:(BOOL)animated
+{
+    [super viewWillAppear:animated];
+
+    self.tableView.tableFooterView = [[UIView alloc] initWithFrame:CGRectZero];
```

Comment:
I wondered the same thing a couple of months ago when I saw this sprinkled everywhere in our codebase.  I googled a bit and learned on StackOverflow that this is a hack to work around some bug in UITableView.  I can't remember what the bug is, but I've been propagating this fix as I make new UITableViews.